### DISCUSSION TITLE
AutoARIMA fit takes a long time

### DISCUSSION THREAD

**Discussion by matsuobasho** [Nixtla/statsforecast#864] [General]
I'm just starting with `statsforecast`.

I have univariate time series data with a frequency of 15minutes.  I subsetted it to just 3 months of data (11K records).

```python
sf = StatsForecast(
    models = [AutoARIMA(season_length = 96)],
    freq = '15min'
)

sf.fit(df_comb[df_comb.ds>='2022-09-01'])
```

It ran for 87 minutes on my machine before I killed it.  Is this normal?

**Comment by jmoralez** [comment]
The ARIMA model doesn't scale very well with the season length, [Rob Hyndman suggests creating fourier terms instead](https://robjhyndman.com/hyndsight/longseasonality/) to model the seasonal patterns, you can find an example on how to do that for statsforecast [here](https://nixtlaverse.nixtla.io/statsforecast/docs/how-to-guides/sklearn_models.html#generating-features)

**Comment by matsuobasho** [comment]
Thanks, @jmoralez  for the quick help.  Will try fourier terms as per your suggestion.
